charts/tidb-cluster/templates/monitor-service.yaml

```diff
@@ -27,6 +27,30 @@ spec:
     app.kubernetes.io/component: monitor
 {{- end }}
 ---
+{{- if .Values.monitor.reloader.create }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "cluster.name" . }}-reloader
```

Comment:
Should add `monitor` in the service name.

Reply:
Done